api-consumption/src/main/java/org/zalando/nakadi/controller/SubscriptionStreamController.java

```diff
@@ -208,10 +206,10 @@ private StreamingResponseBody stream(final String subscriptionId,
                                          final Client client,
                                          final StreamParameters streamParameters) {
 
-        final Span requestSpan = TracingService.getActiveSpan();
-        requestSpan.setOperationName("stream_events")
+        TracingService.setOperationName("stream_events")
                 .setTag("subscription.id", subscriptionId);
 
+        final Span requestSpan = TracingService.getActiveSpan();
```

Comment:
Nice! 
Just wanted to mention, that from these 3 lines it is very much clear what is happening - we are settings properties for current span and then storing it for referencing later. Much much less time is needed to follow the code.